Makefile

```diff
@@ -37,8 +38,8 @@ lint: format fix clippy ## Runs all linting tasks at once (Clippy, fixing, forma
 # --- docs ----------------------------------------------------------------------------------------
 
 .PHONY: doc
-doc: ## Generates & checks documentation 
-	$(WARNINGS) cargo doc --all-features --keep-going --release
+doc: ## Generates & checks documentation
```

Comment:
FYI @bobbinth the miden base does not pass the doc command + doc command is not in CI should we add it and make the repo pass?

Reply:
Yes - that would be good. But if it adds a lot of work, I'd do it in a different PR.